src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceService.scala

```diff
@@ -13,32 +13,36 @@ import scala.concurrent.{ExecutionContext, Future}
 /**
   * Created by mbemis on 5/22/17.
   */
-class ResourceService(val accessPolicyDAO: AccessPolicyDAO, val directoryDAO: DirectoryDAO, val googleDomain: String)(implicit val executionContext: ExecutionContext) extends LazyLogging {
+class ResourceService(private val resourceTypes: Map[ResourceTypeName, ResourceType], val accessPolicyDAO: AccessPolicyDAO, val directoryDAO: DirectoryDAO, val googleDomain: String)(implicit val executionContext: ExecutionContext) extends LazyLogging {
+  def getResourceTypes(): Future[Map[ResourceTypeName, ResourceType]] = {
```

Comment:
this function and the next both return Future even though it is not necessary at this time. Returning Future is consistent with every other function plus I expect we will starting pulling this information from ldap at some point in the .... Future.